### DISCUSSION TITLE
Why was MDX deprecated in Storybook 7 ?

### DISCUSSION THREAD

**Discussion by maxime4000** [storybookjs/storybook#21018] [Help]
Hi, just want to know what are the reason MDX format was dropped in Storybook 7?
Personally, I prefer MDX Style of Docs than CSF Docs. Not saying that CSF Docs is bad, but generally when I had to use OSS package that was offering CSF Docs or a similar package with MDX Docs, I had always had better experience with the package with a MDX Docs. Imo, MDX is the superior format for Docs, so I don't understand why it's deprecated. 

Tbh, I'm not even sure if it's MDX generally or it's "MDX only" stories that are deprecated. The docs doesn't explain much other than this is deprecated in Storybook 7, here a script to migrate to CSF. So maybe I can get the answer here, is MDX dead ? Is MDX+CSF alive ? Maybe the docs could be update to make this more clear ? 

Thanks for your time. :)

**Comment by bodograumann** [comment]
> We heard from a lot of users that they were unsure if they should write stories in MDX files if they were documenting them (they didn’t have to) and what the benefits were (it wasn’t clear there were that many). In order to reduce scope and support future use cases, we decided to deprecate that feature.

Source: https://storybook.js.org/blog/storybook-7-docs/

I also feel like tooling for stories in mdx was never on-par. Personally I liked having the documentation in one place though...
